kong/observability/logs.lua

```diff
@@ -120,13 +124,13 @@ local function get_request_log_buffer()
 end
 
 
-function _M.maybe_push(stack_level, log_level, ...)
+function _M.maybe_push(stack_level, attributes, log_level, ...)
   -- WARNING: do not yield in this function, as it is called from ngx.log
 
   -- Early return cases:
 
   -- log level too low
-  if configured_log_level() < log_level then
+  if log_level and configured_log_level() < log_level then
```

Comment:
Does this mean that if the logs are recorded through `kong.telemetry.log`, their log level will be ignored and they will be forced to be logged?

Reply:
correct. If `kong.telemetry.log` is called the log entry is generated and pushed, there is no concept of "log level" in that case.